lxd/instances.go

```diff
@@ -438,7 +438,7 @@ func instancesShutdown(s *state.State, instances []instance.Instance) error {
 					// If DB was available then the instance shutdown process will have set
 					// the last power state to STOPPED, so set that back to RUNNING so that
 					// when LXD restarts the instance will be started again.
-					inst.VolatileSet(map[string]string{"volatile.last_state.power": "RUNNING"})
+					_ = inst.VolatileSet(map[string]string{"volatile.last_state.power": "RUNNING"})
```

Comment:
We should log this error as a warning.

Reply:
Oh actually don't worry about it here, as in most cases where this will not happen its because the if statement above it will indicate there's no DB available and thus won't log the warning anyway.